data/json/field_type.json

```
@@ -935,6 +935,66 @@
     "phase": "gas",
     "display_field": true
   },
+  {
+    "id": "fd_hot_air_sauna",
+    "type": "field_type",
+    "intensity_levels": [
+      { "name": "hot air sauna", "sym": "&", "convection_temperature_mod": 65 },
+      { "color": "yellow", "convection_temperature_mod": 68 },
+      { "color": "red", "convection_temperature_mod": 71 }
+    ],
+    "decay_amount_factor": 5,
+    "percent_spread": 100,
+    "outdoor_age_speedup": "100 minutes",
+    "priority": -1,
+    "half_life": "50 minutes",
+    "phase": "gas",
+    "display_field": true
+  },
+  {
+    "id": "fd_migo_atmosphere",
+    "type": "field_type",
+    "intensity_levels": [
+      {
+        "name": "foul-smelling air",
+        "sym": "8",
+        "dangerous": true,
+        "effect_id": "migo_atmosphere",
```

There is a slightly different syntax for fields effects after #35638 is merged:

```suggestion
        "effects": [
          {
            "effect_id": "migo_atmosphere",
            "body_part": "MOUTH",
            "intensity": 1,
            "min_duration": "5 seconds",
            "max_duration": "10 seconds",
            "immune_inside_vehicle": true
          }
```